test/integration/oracle.js

```diff
@@ -81,4 +82,31 @@ describe('Oracle', function () {
     const response = await query.pollForResponse({ attempts: 2, interval: 1000 })
     response.decode().toString().should.be.equal(queryResponse)
   })
+
+  describe('Oracle query fee settings', async () => {
+    let oracleWithFee
+    const queryFee = 24000
+    const account = generateKeyPair()
+
+    before(async function () {
+      await client.spend('1' + '0'.repeat(15), account.publicKey)
```

Comment:
```suggestion
      await client.spend(1e15, account.publicKey)
```
won't it work the same?